src/app/app-platform/ContentAppPlatform.h

```diff
@@ -33,6 +33,8 @@ using chip::app::Clusters::ApplicationBasic::CatalogVendorApp;
 using chip::Controller::CommandResponseFailureCallback;
 using chip::Controller::CommandResponseSuccessCallback;
 
+using BindingListTypeInfo = chip::app::Clusters::Binding::Attributes::Binding::TypeInfo::Type;
```

Comment:
```suggestion
using BindingListType = chip::app::Clusters::Binding::Attributes::Binding::TypeInfo::Type;
```
This is not a typeinfo in any meaningful sense.

Reply:
fixed